docs/getting-started/joining-matrix-channels.md

```diff
@@ -0,0 +1,62 @@
+# Joining the Decred community at Matrix
+
+---
+
+## What is Matrix
+
+Matrix is a secure, open messaging platform popular with Decred contractors. With the help of bridge software Matrix users can communicate with users on other Decred platforms such as Discord. Matrix offers first party registration; no identifying information or third party service is required to sign up. Users should be aware however that some features, such as password reset, do require an email address. For more on Matrix’s security features and available clients, see the Matrix project homepage.
+
+To join Matrix, you’ll need to choose a client and a homeserver. Matrix supports numerous clients and homeservers, but for first-time users, we recommend the most well-supported Matrix client, Riot, and Decred’s homeserver. Using the Decred homeserver makes it easier to find Decred chat rooms, puts `decred.org` in your ID, and allows you to continue messaging should Decred’s homeserver become disconnected with other homeservers.
+
+---
+
+## Creating your account
+
+Download [Riot](https://about.riot.im/), an open-source Matrix client.
```

Comment:
We should mention that downloading Riot is optional, it can also be used on the web either at https://riot.im/ or https://chat.decred.org/

Reply:
Changed it to "Access Decred's server using Riot..." and added both links to the web browser option.